src/main/java/com/gluonhq/richtext/model/Piece.java

```diff
@@ -23,7 +23,7 @@ public Piece(final PieceTable source, final BufferType bufferType, final int sta
         this.start = start;
         this.length = Math.max(length, 0);
         this.source = Objects.requireNonNull(source);
-        this.decoration = decoration == null ? TextDecoration.builder().build() : decoration;
+        this.decoration = decoration == null ? TextDecoration.builder().buildDefault() : decoration;
```

Comment:
I'll use something like:
`TextDecoration.builder().default().build()` instead?

Reply:
`default` is a keyword :)

Reply:
Right... `TextDecoration.builder().reset().build()` or `TextDecoration.builder().presets().build()`?